docs/user-guides/plugin/plugin-tests.md

```diff
@@ -124,11 +124,29 @@ testFramework:
   imagePullPolicy: IfNotPresent
 ```
 
+6. **Running the Tests**:
+
+> **Important:** Once you have completed all the steps above, you are ready to run the tests. However, before running the tests, ensure that you perform a fresh Helm installation or upgrade of your plugin's Helm release by executing the following command:
+
+```yaml
+# For a new installation
+helm install <Release name> <chart-path>
+
+# For an upgrade
+helm upgrade <Release name> <chart-path>
+```
+
+- After the Helm installation or upgrade is successful, Run the tests against a test Kubernetes cluster (for example, Minikube or Kind) by executing the following command.
```

Comment:
```suggestion
- After the Helm installation or upgrade is successful, run the tests against a test Kubernetes cluster (for example, Minikube or Kind) by executing the following command.
```

Reply:
The tests must run against the same cluster where the chart is installed. How about moving the recommendation to runs this against a test cluster right to the beginning of the section?